WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+
+If you want to cast either spell at a higher level, you must expend a spell slot as normal.
+
+## Magical Studies
+Mages are spellcasters who favor magics involving the cardinal elements of the universe. Students gifted with a keen intellect and unwavering discipline may walk the path of the mage. The magic available to mages is both great and dangerous, and thus is revealed only to the most devoted practitioners.
+
+### Study of Arcane
+Mages of the arcane are diviners of secrets, balancing the webb and flow of incredible mystic energies. These practi-tioners push their magical knowledge to its very limits. Those who master this craft are capable to manipulate spells on a much more primal level.
+
+#### Bonus Cantrip
+*2nd-level Arcane feature* <br>
+When you choose this magical study at 2nd level, you learn the *prestidigitation* cantrip. If you already know this cantrip, you learn a different wizard cantrip of your choice. The cantrip doesn't count against your number of cantrips known.
+
+\columnbreak
+
+#### Arcane Charges
+*2nd-level Arcane feature* <br>
+You gain the ability to conserve and enhance your mage abilities beyond the norm. Whenever you cast a mage spell of 1st level or higher, gain an arcane charge of the equivalent level up to 5th level. Spells slots higher than 5th give a 5th level arcane charge. These charges last until replaced by a higher level charge, used, or 1 minute has passed.
+
+You may expend an arcane charge as a bonus action to grant yourself one of the following effects. This effect lasts until the end of your next turn.
+- A bonus equal to the arcane charge's level to Attack and Damage rolls for your next Mage Spell.
+- A bonus equal to the arcane charge's level to Saving Throws against Magic or Magical Effects. You may use a reaction for this effect instead of a bonus action.
+
+#### Chrono Shift
+*2nd-level Arcane feature* <br>
+After rolling initiative, but before the first creature takes its turn, you can switch your result with the result of another creature that you can see. Once you've done so you can't use this feature again until you finish a long rest.
+
+#### Arcane Brilliance
+*6th-level Arcane feature* <br>
+You gain proficiency in the Arcana skill if you don't already have it. Your proficiency bonus is doubled for any ability check you make with it.
+
+Additionally, you are always under the effects of a *comprehend languages* spell.
+
+#### Presence of Mind
+*10th-level Arcane feature* <br>
+Whenever you create a spell slot from sorcery points you also gain an arcane charge of equivalent level and you can use it as part of the same bonus action.
```

Comment:
Is this the Arcane Charge to be used as part of the same bonus action?

Reply:
Is this clearer?

Whenever you create a spell slot from sorcery points you also gain an arcane charge of equivalent level and you can expend the charge as part of the same bonus action.

Reply:
That is a lot clearer, yep!